src/components/Publish/_validation.ts

```diff
@@ -78,11 +80,23 @@ const validationPricing = {
       (param) => maxDecimalsValidation.test(param?.toString())
     )
     .required('Required'),
-  amountDataToken: Yup.number()
-    .min(50, (param) => `Must be more or equal to ${param.min}`)
-    .required('Required'),
+  amountDataToken: Yup.number().required('Required'),
   amountOcean: Yup.number()
-    .min(50, (param) => `Must be more or equal to ${param.min}`)
+    .test('validator-min-amountOcean', '', function (value) {
+      const minValue =
+        this.parent.price > 0
+          ? new Decimal(this.parent.price)
+              .mul(this.parent.weightOnOcean)
+              .mul(10)
+              .toDecimalPlaces(MAX_DECIMALS)
+              .toString()
+          : initialValues.pricing.amountOcean.toString()
+      return value < parseInt(minValue)
+        ? this.createError({
+            message: `Must be more or equal to ${minValue}`
```

Comment:
Changing this message might be already high impact without starting to show errors on datatoken field, which we wanted to prevent as it invites editing of a field is not editable.

Not sure how feasible but something like this:

`Must be more or equal to ${minValue}, as at least ${minValueDatatoken} ${datatokenSymbol} are required for this pool to work properly.`